etc/init.d/contextBroker.centos

```diff
@@ -91,7 +91,9 @@ broker_start()
 {
     printf "%-50s" "Starting ${NAME}..."
     if [[ -x ${EXECUTABLE} ]]; then
-        su $BROKER_USER -c "${EXECUTABLE} ${BROKER_OPTS} &> /dev/null "
+        # preserving existing error log (if it already exists)
+        su $BROKER_USER -C "mv ${BROKER_LOG_DIR}.${EXECUTABLE}.err ${BROKER_LOG_DIR}.${EXECUTABLE}.err.old 2> /dev/null || True"
```

Comment:
The only reason to use "|| true" is to avoid that the entire statement returns != true because if it dies, the execution ends there.   make works this way. contextBroker.centos is input to ... what executable?  Might work the way I describe. If so, the "|| true" makes sense, if not ... it doesn't.